MvvmCross/Platforms/WinUi/Views/MvxApplication.cs

```diff
@@ -101,6 +111,52 @@ protected virtual void RegisterSetup()
         {
 
         }
+
+        [DllImport("user32.dll")]
```

Comment:
Not super well versed in WinUI world, but is it really necessary to use PInvoke to manage windows?

Reply:
It doesn't look like it's required. I have modified the MultieWindowViewPresenter to intercept the closing there. This only works thought it the Application.Current is of the type MvxApplication.